package.json

```diff
@@ -61,6 +62,10 @@
     "@dpc-sdp/ripple-timeline": "file:packages/Molecules/Timeline",
     "@dpc-sdp/ripple-updated-date": "file:packages/Organisms/UpdatedDate",
     "@dpc-sdp/ripple-whats-next": "file:packages/Molecules/WhatsNext",
+    "@dpc-sdp/ripple-close-button": "file:packages/Atoms/CloseButton",
+    "@dpc-sdp/ripple-map-indicator": "file:packages/Atoms/MapIndicator",
+    "@dpc-sdp/ripple-map": "file:packages/Molecules/Map",
+    "@dpc-sdp/ripple-wifi-map": "file:packages/Organisms/WifiMap",
```

Comment:
Open to thoughts @richorama @jasondsouza but I had thought the WifiMap can just live in vic-gov-au, but the reusable Map becomes part of Ripple

Reply:
happy to remove the code!

Reply:
removed.